packages/tradershub/src/components/CFDSection/CFDContent/CFDContent.tsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import { useIsEuRegion } from '@deriv/api';
 import { CTraderList, MT5PlatformsList, OtherCFDPlatformsList } from '../../../features/cfd/components';
+import useRegionFlags from '../../../hooks/useRegionFlags';
 import { TradingAppCardLoader } from '../../Loaders';
+import { useUIContext } from '../../UIProvider';
 
 const CFDContent = () => {
-    const { isEU, isSuccess } = useIsEuRegion();
+    const { isSuccess: isEURegionAccessible } = useIsEuRegion();
+    const { getUIState } = useUIContext();
 
-    if (!isSuccess)
+    const activeRegion = getUIState('region');
```

Comment:
better to call this `regulation` 
```suggestion
    const activeRegion = getUIState('regulation');
```